importer.js

```diff
@@ -80,12 +90,21 @@ class Importer {
 
     this.config.csv.columns = (cols) => { // callback for checking columns names in csv
       Object.keys(this.fieldSchema).forEach(key => {
-        if(cols.indexOf(this.namesMapping[key]) < 0) // if key doesn't exist in cols array
-        {
-          console.error('Error: there is no column named ' + this.namesMapping[key] + ' in ' + inputFile);
-          console.error('column names: ' + cols);
-          process.exit(errors.ERROR_BAD_CONFIG_FORMAT);
+        if(typeof this.namesMapping[key] === 'object') { // if 'from' field is an array - checking each of them
```

Comment:
Can you please explain how it works now. 
btw https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Let's try to keep lines shot. The comment would be better move to line above.

Reply:
Oh, I don't know why I didn't find this method earlier, thanks.

It works just like earlier. Just added checking if there is each of array elements in csv-header if "from" is an array.